libcontainer/cgroups/stats.go

```diff
@@ -146,6 +146,17 @@ type HugetlbStats struct {
 	Failcnt uint64 `json:"failcnt"`
 }
 
+type RdmaEntry struct {
+	Device     string `json:"device,omitempty"`
+	HcaHandles uint32 `json:"hca_handles,omitempty"`
```

Comment:
Should these all be `RDMAEntry` and `HCAHandles` (capital RDMA and HCA)?

Reply:
We have a linter for that now, don't we? Let me try to run it.

Reply:
Not sure it the linter understands all acronyms of course.

Reply:
Right. I agree those should be named as in https://github.com/containerd/console/pull/54

Reply:
This ^^^  concern is still valid.

Reply:
@thaJeztah @kolyshkin Following convention is analogous to upstream `opencontainers/runtime-spec` https://github.com/opencontainers/runtime-spec/blob/master/specs-go/config.go#L347 . My Intention is to keep everything consistent across the ecosystem but i am cool with anything